sample-go-continuous-fuzz.conf

```diff
@@ -0,0 +1,33 @@
+; Example configuration for go-continuous-fuzz.
+
+; The default location for this file is in 
+; ~/.go-continuous-fuzz/go-continuous-fuzz.conf on POSIX OSes,
+; $LOCALAPPDATA/Go-continuous-fuzz/go-continuous-fuzz.conf on Windows,
+; ~/Library/Application Support/Go-continuous-fuzz/go-continuous-fuzz.conf on Mac OS
+; and $home/go-continuous-fuzz/go-continuous-fuzz.conf on Plan9.
+
+; If only one value is specified for an option, then this is also the
+; default value used by go-continuous-fuzz.
+; If the part after the equal sign is empty then go-continuous-fuzz has no default 
+; for this option, and it is considered a required option.
+
+[Project]
+; Git URL of the project to fuzz.
+; project.src-repo =
```

Comment:
I think it would be helpful to show an example value for each config option, so the user has a format they can easily imitate.

See LND's `sample-lnd.conf` for an example of how this can be done.